react/features/always-on-top/AlwaysOnTop.js

```diff
@@ -181,6 +246,38 @@ export default class AlwaysOnTop extends Component<*, State> {
 
     _videoAvailabilityListener: ({ available: boolean }) => void;
 
+    /**
+     * Renders display name and avatar for the on stage participant.
+     *
+     * @returns {ReactElement}
+     */
+    _renderVideoNotAvailableScreen() {
+        const { avatarURL, displayName, isVideoDisplayed } = this.state;
+
+        if (isVideoDisplayed) {
+            return null;
+        }
+
+        return (
+            <div id = 'videoNotAvailableScreen'>
+                {
+                    avatarURL
+                        ? <div id = 'avatarContainer'>
+                            <img
+                                id = 'avatar'
+                                src = { avatarURL } />
+                        </div>
+                        : null
+                }
+                <div
+                    className = 'displayname'
+                    id = 'displayname'>
+                    { `${displayName}` }
```

Comment:
Nit: you don't need template strings (or whatever they're called now) to display the value of a single variable.

Reply:
Yes.... Can't remember writing this .... Will fix it!